pixiedust/display/datahandler/pysparkDataFrameHandler.py

```diff
@@ -134,7 +134,7 @@ def getWorkingPandasDataFrame(self, xFields, yFields, extraFields=[], aggregatio
             workingDF = workingDF.dropna()
         count = workingDF.count()
         if count > maxRows:
-            workingDF = workingDF.sample(False, (float(maxRows) / float(count)))
+            workingDF = workingDF.sample(n=int(maxRows),replace=False)
```

Comment:
workingDF in this file is a spark DataFrame, and the sample method of DataFrame doesn't have a n argument.